### ISSUE TITLE
Github is not a place for political activities.

### ISSUE THREAD

**Issue by alex20465** [stackshareio/dev-tools-for-blm#1]

**Comment by The-Don-Himself** [comment]
Disappointing that you classify racism as a political issue, its a society issue. So unless you mean Github is not a place for social activities...

**Comment by comradekingu** [comment]
It isn't so much the supposed political nature, it lacks

>Misinformation and disinformation - You may not post content that presents a distorted view of reality, whether it is inaccurate or false (misinformation) or is intentionally deceptive (disinformation) because such content is likely to result in harm to the public or to interfere with fair and equal opportunities for all to participate in public life. For example, we do not allow content that may put the well-being of groups of people at risk or limit their ability to take part in a free and open society. We encourage active participation in the expression of ideas, perspectives, and experiences and may not be in a position to dispute personal accounts or observations. We generally allow parody and satire that is in line with our Acceptable Use Polices, and we consider context to be important in how information is received and understood; therefore, it may be appropriate to clarify your intentions via disclaimers or other means, as well as the source(s) of your information.

From https://help.github.com/en/github/site-policy/github-acceptable-use-policies

>2. Content Restrictions
Under no circumstances will Users upload, post, host, execute, or transmit any Content to any repositories that:
…
is discriminatory or abusive toward any individual or group;

This repo is seeking donations for business owners exclusively based on their race.

>use our servers for any form of excessive automated bulk activity (for example, spamming or cryptocurrency mining), to place undue burden on our servers through automated means, or to relay any form of unsolicited advertising or solicitation through our servers, such as get-rich-quick schemes;

With a makeshift website

>Impersonation - You may not seek to mislead others as to your identity by copying another person's avatar, posting content under their email address, using a similar username or otherwise posing as someone else. Impersonation is a form of harassment.

The repository contains no code, or creative endeavor. No ideas, perspectives, experiences or disclaimers are presented.

@The-Don-Himself 

https://www.bjs.gov/content/pub/pdf/ph98.pdf
https://www.pnas.org/content/116/34/16793
https://scholar.harvard.edu/files/fryer/files/fryer_police_aer.pdf
https://onlinelibrary.wiley.com/doi/full/10.1111/1745-9133.12187
https://journals.sagepub.com/doi/abs/10.1177/1098611117732974
https://www.pnas.org/content/116/32/15877
https://ucr.fbi.gov/crime-in-the-u.s/2017/crime-in-the-u.s.-2017/topic-pages/tables/table-42

TL;DR https://www.invidio.us/watch?v=ItopNgRQiuY
